single-page.bs

```diff
@@ -733,6 +734,11 @@ urlPrefix: https://tools.ietf.org/html/
     url: rfc7232#section-2.2; type: http-header; spec: rfc7232; text: last-modified
     url: rfc7234#section-5.2; type: http-header; spec: rfc7234; text: cache-control
 
+#  ************************** INFRA *************************************
```

Comment:
this is unnecessary, bikeshed already has this reference. (Plus it is unnecessary as noted above)